extensions/wikia/Wall/WallThread.class.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 class WallThread {
+	const DB_LIMIT_REPLIES = 500;
```

Comment:
The limit is only the query limit for recursion. All replies will be loaded on the page; the user experience doesn't change.